src/containers/App/index.tsx

```diff
@@ -1,17 +1,203 @@
 import React from "react";
+import { FcBusinessman } from "react-icons/fc";
+import {BrowserRouter as Router, Route, Switch, Link} from "react-router-dom";
+import DashboardRoundedIcon from '@material-ui/icons/DashboardRounded';
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import EditIcon from '@material-ui/icons/Edit';
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import SettingsInputComponentIcon from '@material-ui/icons/SettingsInputComponent';
+import Dashboard from '../Dashboard';
+import Add from '../Add';
+import Edit from '../Edit';
+import View from '../View';
+import Settings from '../Settings';
 
+
+
+const toolbar: React.CSSProperties = {
+      position: "fixed",
+      top: "0",
+      left: "0",
+      width: "100%",
+      background: "#3f51bf",
```

Comment:
We should consider making a constants file that way we can put a variable name to colors.